### PR TITLE
[ADD] Limit write if room not in user rooms

### PR DESCRIPTION
**Pull request by eilst**

### CODE REVIEW

File: project_event/security/project_security.xml [hunk 1]
```diff
@@ -130,14 +119,26 @@
...
-        <field name="groups" eval="[(4, ref('group_project_event_editor')),   (4, ref('group_project_event_manager'))]"/>
+        <field name="groups" eval="[(4, ref('group_project_event_manager'))]"/>
         <field eval="1" name="perm_unlink"/>
         <field eval="1" name="perm_write"/>
         <field eval="1" name="perm_read"/>
         <field eval="1" name="perm_create"/>
     </record>
+
+    <record id="calendar_event_access_project_editor_all" model="ir.rule">
+        <field name="name">Calendar Event all access to editor</field>
```
Comment (rimbendhaou) [inline project_event/security/project_security.xml]: not all access, only read access in this case

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/security/project_security.xml (hunk 1 above)
Comment (eilst) [reply project_event/security/project_security.xml]: 1=1 i mean, access to all records

Comment (eilst) [reply project_event/security/project_security.xml]: For this kind of comment please just comment, not request change; so if you are not available, we can still merge it  if someone else does the review

**Review by rimbendhaou — APPROVED** [review]

File: project_event/security/project_security.xml (hunk 1 above)
Comment (rimbendhaou) [inline project_event/security/project_security.xml]:
```suggestion
        <field name="name">All Calendar Event read access to editor</field>
```

**Pull request merged**